### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by startupandrew**
@katowulf -- please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ the `$firebase` service.
 
 ## Getting Started with Firebase
 
-AngularFire requires Firebase in order to sync data. You can [sign up here for a free
+AngularFire uses the Firebase database to store and sync data. You can [sign up here for a free
```
Comment (katowulf) [inline README.md]: It also uses Firebase Auth. Should we state that here?

**Comment by katowulf** [comment]
Nice! I don't think I've ever assigned you anything before. I feel powerful... yes... what can I give James to do?

**Comment by startupandrew** [comment]
Updated. How's this?

**Comment by startupandrew** [comment]
Ok -- I can't imagine my readme updates broke travis : ) Is there a transient error in our tests?

**Comment by jwngr** [comment]
Protractor tests are pretty flaky. Much better than they used to be, but sometimes they randomly fail. I restarted Travis and will babysit it until it passes.

**Comment by jwngr** [comment]
Passing :)

**Comment by startupandrew** [comment]
Thanks Jacob : ) @katowulf -- over to you!

**Pull request merged**